tests/datadotworld/client/test_client.py

```diff
@@ -486,3 +497,7 @@ def test_search(self, api_client, search_api, project_key):
         search_results = api_client.search_resources(query="test")
         assert_that(search_results, has_properties({'count': 1, 'facets': None,
                     'hydrations': None, 'next': None, 'records': []}))
+        
+    def test_create_table(self, api_client, tables_api):
+        result = api_client.create_new_tables('owner','id')
```

Comment:
I think we should input more complete data, for example a `TableCreateOrUpdateRequest` to fully test the function.